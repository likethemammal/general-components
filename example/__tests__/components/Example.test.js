import React from 'react'
import {shallow} from 'enzyme'

import shared from '../../../bin/__tests__/_shared/tests'
import Example from '../../../example/components/Example.unit'


const changeAction = jest.fn()
const component_jsx = <Example
    changeAction={changeAction}
    on={true}
/>

describe('Example', () => {
    // Render a checkbox with label in the document
    const component = shallow(
        component_jsx,
    )

    shared.SNAPSHOTS_SHOULD_MATCH([
        component,
    ])

    shared.SHOULD_CALL_MOCK_FROM_SIMULATE(
        component,
        changeAction,
    )
})