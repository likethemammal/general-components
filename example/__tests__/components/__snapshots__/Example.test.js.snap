// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Snapshot 0 should match 1`] = `
<div
  onClick={[Function]}
/>
`;
